src/data/panel.ts

```diff
@@ -9,27 +9,26 @@ export const getDefaultPanelUrlPath = () =>
 export const getDefaultPanel = (panels: HomeAssistant["panels"]) =>
   panels[localStorage.defaultPage] || panels[DEFAULT_PANEL];
 
-export const getPanelTitle = (hass: HomeAssistant): string | undefined => {
+export const getPanelTitle = (hass: HomeAssistant): string | null => {
   if (!hass.panels) {
-    return;
+    return null;
```

Comment:
Why return `null` instead of `undefined`?

Reply:
I'm not the developer of this but I was curious about this too and for example Sonar suggests in its rules to use always null.

Looking more articles in the web "The main difference is that undefined represents the value of a variable that wasn’t yet initialized, while null represents an intentional absence of an object."

To me it seems intentional here that he wants to return "nothing", so it seems ok to me return null.

Reply:
`panel.title` has the type `string | null` which caused the ts error.

I've added additional logic to revert this change